frontend/src/components/OdhDocCard.tsx

```diff
@@ -87,6 +87,7 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFavorit
           onClick={fireResourceAccessedEvent(odhDoc.metadata.name, odhDoc.spec.type)}
           target="_blank"
           rel="noopener noreferrer"
+          data-testid={`view-documentation ${odhDoc.metadata.name}`}
```

Comment:
If the card already has a unique test ID we don't need to qualify these here. It is sufficient to chain selectors first with to the card and then to the button.
```suggestion
          data-testid="view-documentation"
```

Reply:
I will update it.